src/libraries/System.Text.Json/gen/JsonSourceGenerator.Parser.cs

```diff
@@ -40,9 +40,10 @@ private sealed class Parser
             private const string JsonPropertyNameAttributeFullName = "System.Text.Json.Serialization.JsonPropertyNameAttribute";
             private const string JsonPropertyOrderAttributeFullName = "System.Text.Json.Serialization.JsonPropertyOrderAttribute";
             private const string JsonSerializerContextFullName = "System.Text.Json.Serialization.JsonSerializerContext";
-            private const string JsonSerializerAttributeFullName = "System.Text.Json.Serialization.JsonSerializableAttribute";
             private const string JsonSourceGenerationOptionsAttributeFullName = "System.Text.Json.Serialization.JsonSourceGenerationOptionsAttribute";
 
+            internal const string JsonSerializableAttributeFullName = "System.Text.Json.Serialization.JsonSerializableAttribute";
```

Comment:
single deviation.  In 7.0 line this was renamed (from Serializer -> Serializable).  I kept the 7.0 name (more correct), requiring teh update to line 243 below.